manifests/lge_d802.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+<remote name="enes" 
```

Comment:
You don't need to add a new remote with name "enes" since there already is one for "los" which points to LineageOS.